common/terrain/01_terrain.txt

```diff
@@ -1,7 +1,7 @@
-﻿#The materials referenced in this file are the materials in the file gfx\map\terrain\materials.settings and
+﻿#The materials referenced in this file are the materials in the file gfx/map/terrain/materials.settings and
 #are used when regenerating the province terrain based on what terrain materials have been painted where.
 
-#The materials referenced in this file are the materials in the file gfx\map\terrain\materials.settings and
+#The materials referenced in this file are the materials in the file gfx/map/terrain/materials.settings and
```

Comment:
this change is to make check-spelling v0.0.21 happy. w/ the next version it wouldn't be necessary, but in general you use `/`s anyway, so it's a reasonable change.